force-app/main/default/classes/CON_ContactMerge_TEST.cls

```diff
@@ -409,6 +409,184 @@ public class CON_ContactMerge_TEST {
                 'account should not have changed.');
     }
 
+    @IsTest
+    private static void mergingTwoContactsWithActiveRDSustainersChangeToActive() {
+        //skip the test if Advancement is installed
+        if(ADV_PackageInfo_SVC.useAdv()) return;
+        RD2_EnablementService_TEST.setRecurringDonations2Enabled();
+
+        CON_ContactMerge_CTRL controller = getController();
+        List<Contact> testContacts = getContacts(1);
+        testContacts.add(getUniqueContact());
+        setupMockDataForMerge(controller, testContacts);
+
+        Contact losingContact = testContacts[0];
+        Contact winningContact = testContacts[1];
+
+        insert new List<npe03__Recurring_Donation__c> {
+            TEST_RecurringDonationBuilder.constructEnhancedBuilder()
+            .withDefaultValues()
+            .withStatusActive()
+            .withContact(losingContact.Id)
+            .withAmount(1)
+            .build(),
+
+            TEST_RecurringDonationBuilder.constructEnhancedBuilder()
+            .withDefaultValues()
+            .withStatusClosed()
+            .withContact(winningContact.Id)
+            .withAmount(1)
+            .build()
+        };
+
```

Comment:
Can we add a few assertions here to make sure our original data is what we expect? winningContact will be Sustainer = "Former", losingContact will be Sustainer = "Active".